README.md

```diff
@@ -390,7 +389,10 @@ public static class ConfluentCloudTrigger
 site.
 **ConfluentCloudPassword**: is you API secret, obtained from the Confluent Cloud web site.
 
-3. Download and set the CA certification location. As described in [Confluent documentation](https://github.com/confluentinc/examples/tree/5.4.0-post/clients/cloud/csharp#produce-records), the .NET library does not have the capability to access root CA certificates.<br>
+
+**NOTE:** Microsoft.Azure.WebJobs.Extensions.Kafka (3.1.0)+ doesn't require CA certification location.
+
+3. (Deprecated) Download and set the CA certification location. As described in [Confluent documentation](https://github.com/confluentinc/examples/tree/5.4.0-post/clients/cloud/csharp#produce-records), the .NET library does not have the capability to access root CA certificates.<br>

```

Comment:
do we want to leave (deprecated) in the documentation? 
i'd rather just take it out, and include in the release notes for this version that CA certs are no longer required.

Reply:
Sure @ryancrawcour I'll do it.

Reply:
Done.